### PR TITLE
Use MINIMUM_CONTRACT_PAYMENT_LINK_JUELS instead of MINIMUM_CONTRACT_PAYMENT

### PR DESCRIPTION
**Pull request by j16r**
Issues a deprecation warning at startup if still using MINIMUM_CONTRACT_PAYMENT

### CODE REVIEW

File: core/cmd/local_client_test.go [hunk 1]
```diff
@@ -96,7 +96,7 @@ func TestClient_RunNodeShowsEnv(t *testing.T) {
 	assert.Contains(t, logs, "LOG_TO_DISK: true")
 	assert.Contains(t, logs, "MIN_INCOMING_CONFIRMATIONS: 1\\n")
 	assert.Contains(t, logs, "MIN_OUTGOING_CONFIRMATIONS: 6\\n")
-	assert.Contains(t, logs, "MINIMUM_CONTRACT_PAYMENT: 0.000000000000000100\\n")
+	assert.Contains(t, logs, "MINIMUM_CONTRACT_PAYMENT_LINK_JUELS: 0.000000000000000100\\n")
```
Comment (samsondav) [inline core/cmd/local_client_test.go:99]: This seems wrong... wouldn't it be 100 Juels?

File: core/store/orm/config.go [hunk 2]
```diff
@@ -1168,8 +1178,20 @@ func (c Config) MinRequiredOutgoingConfirmations() uint64 {
...
 	minimumContractPayment := chainSpecificConfig(c).MinimumContractPayment
-	if c.viper.IsSet(EnvVarName("MinimumContractPayment")) || minimumContractPayment == nil {
+	if c.viper.IsSet(EnvVarName("MinimumContractPayment")) {
 		return c.getWithFallback("MinimumContractPayment", parseLink).(*assets.Link)
+
+		// TODO: Remove when implementing
+		// https://app.clubhouse.io/chainlinklabs/story/8096/fully-deprecate-minimum-contract-payment
+	} else if c.viper.IsSet("MINIMUM_CONTRACT_PAYMENT") {
+		str := c.viper.GetString("MINIMUM_CONTRACT_PAYMENT")
+		value, ok := new(assets.Link).SetString(str, 10)
```
Comment (samsondav) [inline core/store/orm/config.go:1188]: Doesn't this change the behaviour from what it used to mean?

**Review by samsondav — COMMENTED** [review]
Don't forget to update the configuration page

https://docs.chain.link/docs/configuration-variables/

File: core/cmd/local_client_test.go (hunk 1 above)
Comment (j16r) [reply core/cmd/local_client_test.go:99]: Ahh yep, the display functions will have to be updated.

File: core/store/orm/config.go (hunk 2 above)
Comment (j16r) [reply core/store/orm/config.go:1188]: I believe this is doing exactly what it was doing before, i.e. this is just the body of parse parseLink.

**Review by samsondav — APPROVED** [review]

**Pull request merged**